lib/helpers/svgo-merge.js

```diff
@@ -0,0 +1,26 @@
+const { merge, mergeWithCustomize } = require('webpack-merge');
+// SVGO doesn't export `resolvePluginConfig` in the default module.
+const { resolvePluginConfig } = require('svgo/lib/svgo/config');
```

Comment:
This is gross, but I couldn’t figure out a better way to do it because svgo doesn’t properly export this.